src/server/admin/server/api_server.go

```diff
@@ -19,12 +20,14 @@ import (
 type Env struct {
 	ClusterID string
 	Config    *pachconfig.Configuration
+	PFSServer pfs.APIServer
```

Comment:
At some point, we should migrate all these grpc servers to be clients.  But this seems like the right interface for now.